experimental/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ All notable changes to experimental packages in this project will be documented
     * (internal) OTLPExporterBrowserBase: `RequestType` has been replaced by a `ResponseType` type-argument
     * (internal) OTLPExporterNodeBase: `ServiceRequest` has been replaced by a `ServiceResponse` type-argument
     * (internal) the `@opentelemetry/otlp-exporter-proto-base` package has been removed, and will from now on be deprecated in `npm`
+* fix(instrumentation)!: make config object required in base instrumentation [#4695](https://github.com/open-telemetry/opentelemetry-js/pull/4695): @blumamir
```

Comment:
I think this is not making it required as much as providing a default value. User still doesn't have to provide a config object if they don't want to

Reply:
Updated the text 👍